.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,60 @@
+name: Bug report
+description: Create a report to help us improve
+labels: ["bug", "triage"]
+assignees:
+  - wizard-28
+  - D-Brox
+body:
+  - type: checkboxes
+    id: confirmation
+    attributes:
+      label: Are you sure this is an issue with LibPacstall and not with the program that uses this library?
+      options:
+        - label: Yes, I am sure this is an issue with LibPacstall
+          required: true
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      placeholder: Tell us what happened!
+    validations:
+      required: true
+  - type: textarea
+    id: steps
+    attributes:
+      label: To reproduce
+      placeholder: 1. Write step by step what you did
+    validations:
+      required: true
+  - type: textarea
+    id: expectation
+    attributes:
+      label: Expected behavior
+    validations:
+          required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      placeholder: Attach or link to screenshots here.
+    validations:
+          required: true
+  - type: input
+    id: os
+    attributes:
+      label: OS version
+      description: Provide your OS version.
+      placeholder: Ubuntu 20.04
+    validations:
+      required: true
+  - type: input
+    id: application
+    attributes:
+      label: Application
+      description: Provide the name of the application you've used LibPacstall with and it's version.
+      placeholder: Pacstall 1.7.2 (develop)
```

Comment:
```suggestion
      placeholder: LibPacstall 0.1.0
```

Reply:
People won't use the library with the library though. See the description.

Reply:
Ok, true, but we need the version of libpacstall too

Reply:
Right how about something like this:

---

# LibPacstall version

What LibPacstall version are you using?

You can find the LibPacstall version with:

```console
python3 -c "import libpacstall; print(libpacstall.__version__)"
```

*Input box here*

Reply:
I'd use:
You can find the LibPacstall version with `python3 -c "import libpacstall; print(libpacstall.__version__)"`
In one single line.
But ye, seem good to me

Reply:
@D-Brox my one would look like this:

![](https://0x0.st/oq_d.png)

If we put it in one line we would loose the syntax highlighting.

Reply:
Resolved in https://github.com/pacstall/libpacstall/pull/18/commits/e68192cbe191a6a20afb0f8c84348e16f53629f4